sql/moz-fx-data-shared-prod/acoustic/contact/metadata.yaml

```diff
@@ -0,0 +1,16 @@
+friendly_name: Adjust contact data
```

Comment:
@kik-kik Small typo to point out (sorry I'm late), I think these are all "Acoustic"-related tables and not "Adjust"-related tables!